features/analyze.feature

```diff
@@ -0,0 +1,53 @@
+Feature: analyze
+    The primary user-facing command. This command will delegate to other freshli activities to accomplish its work. It will manage work queues to enable parallelization.
+
+    ```
+    freshli [global options] analyze [command options] <repo url>|<local dir>
+    ```
+
+    With no options specified, performs analysis locally, and then sends the results to the Freshli web app so that the results can be viewed at at URL that will be provided in the command output.
+
+
+    NOTE: These scenarios are not complete nor properly tested because at the time of writing not everything is done. So the entire chain can't be tested yet.
```

Comment:
What hasn't been done yet? We'll need to create issues for them, and also create an issue to ensure that these scenarios are fleshed out.

Reply:
Also... why wouldn't it make sense to complete everything as part of this pull request?

Reply:
I'll make sure to create the issue for it. 
Why it doesn't make sense to complete everything is that it depends on some other issues that aren't done yet:
https://github.com/corgibytes/freshli-cli/issues/205
https://github.com/corgibytes/freshli-cli/issues/208

I'm going to check the entire flow and see if we are missing any connections and make issues for it. After that's done we can update these features.

Reply:
I've added issues and linked them to the epic here: 
https://github.com/corgibytes/freshli-cli/issues/135#issuecomment-1241804496
There's also an updated image of the happy path flow to show what we are missing because I lost the overview entirely and misguided Dona twice now.
https://github.com/corgibytes/freshli-cli/issues/135#issuecomment-1241805801